torch/_inductor/graph.py

```diff
@@ -209,6 +209,8 @@ def __init__(
         ] = (
             []
         )  # This is the linemap used by the profiler to mark custom compiled kernels getting run
+        # Used if lowering encounters cases where cudagraphs are not supported
+        self.cudagraphs_okay = True
```

Comment:
nit: can we rename this to `disable_cudagraphs` ? this bool is not the the only condition to determine if we can cudagraph